### PR TITLE
fix: update broken plugin-server deployment script

### PR DESCRIPTION
**Pull request by yakkomajuri**
https://github.com/PostHog/posthog/pull/9844 broke self-hosted deployments by removing the KAFKA_HOSTS default. The regex used here does not work on Alpine Linux, so by removing the default `KAFKA_HOSTS` ended up being set to `''`.

### CODE REVIEW

File: bin/plugin-server
Comment (fuziontech) [inline bin/plugin-server:27]: If we do want to fix this regex it should be:
`sed -E "s/kafka((\+ssl)?):\/\///g"`

**Review by hazzadous — APPROVED** [review]

**Review by hazzadous — APPROVED** [review]

File: bin/plugin-server
Comment (hazzadous) [reply bin/plugin-server:27]: This wasn't working before, so merging this in, will unblock release and if this is needed somewhere we can follow up

**Pull request merged**
